main.js

```diff
@@ -0,0 +1,23 @@
+let cnt = 0
+document.getElementById('listen').addEventListener('click',() => {
+    if(localStorage.getItem('counter'))
+    {
+        val = parseInt(localStorage.getItem('counter'));
+        val += 1
+        localStorage.setItem('counter',val)
+    }
+    else{
+        val = 1
+        localStorage.setItem('counter',val)
+    }
+    
+    const count = document.getElementById('count')
+    count.innerHTML = val
```

Comment:
let us avoid using innerHTML, it has a lot of security vulnerabilities.
Better use innerText instead